tests/test_dataserializer.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import Dict
+from gsy_framework.data_serializer import DataSerializer
+
+
+class TestCompressDecompressDict:
+    """Test compress and decompress dictionaries"""
+
+    @staticmethod
+    def test_compress() -> bytes:
+        """Assert if data is bytes type"""
+        data = {'key1': 'value1', 'key2': 'value2'}
+        compressed_data = DataSerializer.encode_and_compress_dict(data)
+        assert type(compressed_data) == bytes
```

Comment:
Nit: I think this test is enough for the moment, because I don't expect this class to change much in the future. But I'd like to show you one shortcoming of this.
Here, we're testing that the `encode_and_compress_dict()` method will return an object of type `bytes`. However, this doesn't guarantee that the bytes object will actually be the compressed version of the input dictionary. Suppose I modify the `encode_and_compress_dict()` method and just return a random byte object:
```py
    @staticmethod
    def encode_and_compress_dict(uncompressed_dict: Dict) -> bytes:
        return bytes("some string", "utf-8")
```
This test will still succeed, even though the method is now clearly not doing what we expect.

I think it could be tricky to fix this because we're dealing with bytes, but it's still worth knowing.